### BIG differences. This is diff of wrong files,
### CLUS "Function CHARACTER" and CLHS topic selection http://clhs.lisp.se/Body/a_ch.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:character.txt	2017-04-27 22:04:34.000000000 +0100
+++ saved-pages-clus/cl:functions:character.txt	2017-04-27 22:04:34.000000000 +0100
@@ -1,4 +1,37 @@
-CHARACTER
-Please select which reference to CHARACTER you intended
+Function CHARACTER

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Syntax
+ character character → denoted-character
+Arguments and Values
+ character - a character designator.
+ denoted-character - a character.
+Description
+Returns the character denoted by the character designator.
+Examples
+(character #\a)
+→
+#\a
+(character "a")
+→
+#\a
+(character 'a)
+→
+#\A
+(character '\a)
+→
+#\a
+(character 65.)
+✖
+ERROR 65 is not a character.
+(character 'apple)
+✖
+ERROR APPLE is not a character.
+Affected By
+None.
+Exceptional Situations
+Should signal an error of type type-error if object is not a character designator.
+See Also
+Function COERCE
+Notes
+(character object) ≡ (coerce object 'character)
+\
\ No newline at end of file
